stories/AttributeFilter.tsx

```diff
@@ -31,3 +36,72 @@ storiesOf('AttributeFilter', module)
             />
         </div>
     ));
+
+const attributeElementsProps: IAttributeElementsProps = {
+    // identifier: '3.df',
+    uri: '/gdc/md/storybook/obj/3.df',
+    projectId: 'storybook',
+    options: {
+        limit: 20
+    },
+    children: ({
+        validElements,
+        loadMore,
+        isLoading,
+        error
+    }: any) => {
+        const {
+            offset = null as any,
+            count = null as any,
+            total = null as any
+        } = validElements ? validElements.paging : {};
```

Comment:
If you added `paging` parameter to the children function, this footprint would be much easier.